src/app/models/user-details.model.ts

```diff
@@ -2,6 +2,7 @@ export interface UserInfo {
   id: string;
   forename: string;
   surname: string;
+  name?: string;
```

Comment:
We should have a chat about this. If we can avoid it, I'd really like to, because it's going to cause a lot of confusion in future I imagine this is being driven by the fact that "CaseMessage" has only a name field. How about if we adopt the convention of using the surname field for name?

Reply:
Property name is currently not required.  It has been removed